feat/Add AI Prompt guard plugin how-to (#2047)
Kong/developer.konghq.com: app/_includes/prereqs/mistral.md

> 1. [Create a Mistral account](https://mistral.ai/signup).

Reviewer: ```suggestion
1. [Create a Mistral account](https://console.mistral.ai/).
```

> 2. [Get your API key](https://mistral.ai/account/api-keys).

Reviewer: ```suggestion
2. [Get your API key](https://docs.mistral.ai/getting-started/quickstart/#account-setup).
```

> 3. Create a decK variable with the Mistral API key:

Reviewer: ```suggestion
3. Export a decK environment variable with the Mistral API key:
```
